test/test_front_matter_defaults.rb

```diff
@@ -27,6 +27,30 @@ class TestFrontMatterDefaults < JekyllUnitTest
     end
   end
 
+  context "A site with full front matter defaults (glob)" do
+    setup do
+      @site = fixture_site({
+        "defaults" => [{
+          "scope"  => {
+            "path" => "contacts/*.html",
```

Comment:
Well, it would, though it would definitely take more custom logic that using `Dir#glob` and then there would be question why `EntryFilter` treats `*.html` differently